README.md

```diff
@@ -138,8 +159,7 @@ Note that in the following instructions, we assume your catkin workspace (in whi
    catkin build -w ~/catkin_ws --cmake-args -DCMAKE_BUILD_TYPE=Release
    ```
 
-1. Source the catkin setup.bash from your catkin workspace.
-
+1. Source the catkin setup.bash from your catkin workspace:
    ```bash
    source ~/catkin_ws/devel/setup.bash
```

Comment:
How often does this need to be done? Ie does it need to be added to the bash startup scripts?

Reply:
yes, you need to do it with every new shell